manifests/e/erickutcher/httpdownloader/1.0.6.5/erickutcher.httpdownloader.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: erickutcher.httpdownloader
+PackageVersion: 1.0.6.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: HTTP_Downloader.exe
+Commands:
+- http-downloader
```

My mistake on the previous suggestion - I missed that it was nested in a zip and that just adding `commands` wouldn't work 🤦 
```suggestion
NestedInstallerFiles:
- RelativeFilePath: HTTP_Downloader.exe
  PortableCommandAlias: http-downloader
```